ibis/backends/polars/__init__.py

```diff
@@ -128,6 +128,8 @@ def _register_failure(self):
         )
 
     def _add_table(self, name: str, obj: pl.LazyFrame | pl.DataFrame) -> None:
+        if isinstance(obj, pl.DataFrame):
+            obj = obj.lazy()
```

Comment:
Seems reasonable. Wondering if we even need to convert things to be lazy? Can we just leave them as is?

Reply:
Oh, sweet! Well, in that case, I will approve this PR!